.github/workflows/ci.yml

```diff
@@ -107,7 +107,9 @@ jobs:
       - uses: actions/checkout@v4
       - uses: Alfresco/alfresco-build-tools/.github/actions/free-hosted-runner-disk-space@v5.33.0
       - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v5.33.0
-      - uses: Alfresco/ya-pmd-scan@v3.0.2
+      - uses: Alfresco/ya-pmd-scan@v4.0.0
+        with:
+          pmd-ruleset-ref-override: "PMD7_3_0"
```

Comment:
This is needed until we can upgrade all the other consumers of `ya-pmd-scan` and drop support for v3 and earlier.

Reply:
So many suppressed warnings removed 🎉  I love it! 😄